src/components/navigation.tsx

```diff
@@ -15,10 +22,18 @@ interface Props {
     faq: string;
     contact: string;
   };
+  languageLabels: {
+    dutch: string;
+    english: string;
```

Comment:
Can we use the locale codes instead, e.g. `nl` rather than `dutch` (so we don't have introduce our own language keys system)?